src/Bowerphp/Repository/GithubRepository.php

```diff
@@ -210,18 +210,17 @@ private function getDepBowerJson($version)
      */
     private function clearGitURL($url)
     {
-        $url = str_replace('git@github.com:', 'github.com/', $url);
-        if (substr($url, 0, 6) == 'git://') {
-            $url = substr($url, 6);
-        }
-        if (substr($url, 0, 8) == 'https://') {
-            $url = substr($url, 8);
-        }
-        if (substr($url, 0, 11) == 'github.com/') {
-            $url = substr($url, 11);
-        } elseif (substr($url, 0, 26) == 'raw.githubusercontent.com/') {
-            $url = substr($url, 26);
+        $partsToClean = array(
+            'git://',
+            'git@github.com:',
+            'https://',
+            'github.com/',
+            'raw.githubusercontent.com/',
+        );
+        foreach($partsToClean as $part) {
```

Comment:
This violates PSR-2. Please run php-cs-fixer on your code before submitting a PR.